modules/rtcpsummary/rtcpsummary.c

```diff
@@ -0,0 +1,122 @@
+/**
+ * @file rtcpsummary.c RTCP summary module
+ * Output RTCP stats at the end of a call if there are any
+ */
```

Comment:
a copyright statement is needed here.

you can for example add Creytiv.com and your own name, if you want